### PR TITLE
Release/0.10.5

### PR DESCRIPTION
**Pull request by kylebaron**
https://github.com/metrumresearchgroup/mrgsolve/milestone/5

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -54,7 +50,9 @@ Suggests:
     rmarkdown,
     yaml,
     knitr, 
-    data.table
+    data.table,
+    pmxTools,
```
Comment (dpastoor) [inline DESCRIPTION:54]: where are pmxTools and spelling being used now? or were they being used but not referenced properly before. I don't see any reference to them in the code?

**Review by dpastoor — COMMENTED** [review]
quick logistic question

File: DESCRIPTION (hunk 1 above)
Comment (kylebaron) [reply DESCRIPTION:54]: pmxTools in tests and I check spelling as part of my CI routine.  I check but I believe sean might have requested spelling in suggests maybe to make sure it got pulled on drone?  But I can remove it so it's not needed except where I build / check locally.

Comment (dpastoor) [reply DESCRIPTION:54]: IMO if spelling is part of purely CI, it shouldn't be in the package dep list - in the same way that we don't add pkgdown/lintr/etc as suggested deps.

**Comment by kylebaron** [comment]
I agree with you.

File: DESCRIPTION (hunk 1 above)
Comment (kylebaron) [reply DESCRIPTION:54]: Yep; I agree.  Just that I'm not the only one making changes to the DESCRIPTION file at this point.

Comment (kylebaron) [reply DESCRIPTION]: Removed spelling from suggests but it made its way into the drone file.  I'll see if it still works.

**Comment by kylebaron** [comment]
Ok; looks like it still worked.

**Review by dpastoor — APPROVED** [review]

**Pull request merged**